bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml

```diff
@@ -24,6 +24,28 @@
 		</tags>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="inverter-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Inverter Temperature</label>
+		<description>Inverter Temperature</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state pattern="%d %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="inverter-workmode-code">
+		<item-type>Number</item-type>
+		<label>Inverter Workmode Code</label>
+		<description>Inverter Workmode Code</description>
+		<state pattern="%d" readOnly="true"/>
+	</channel-type>
+	<channel-type id="inverter-workmode">
```

Comment:
Isn't this workmode actually an inverter status? I guess both words could work but which one is more precise?